build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -202,6 +206,12 @@ egress:
   # The maximum number of Egress IPs that can be assigned to a Node. It's useful when the Node network restricts
   # the number of secondary IPs a Node can have, e.g. EKS. It must not be greater than 255.
   maxEgressIPsPerNode: {{ .maxEgressIPsPerNode }}
+  {{- if ne .snatFullyRandomPorts nil }}
+  # Fully randomize source port mapping in Egress SNAT rules. This has no impact on the default SNAT
+  # rules enforced by each Node for local Pod traffic. By default, we use the same value as for the
+  # top-level snatFullyRandomPorts configuration, but this field can be used as an override.
+  snatFullyRandomPorts: {{ .snatFullyRandomPorts }}
```

Comment:
Do we want to hide the parameter when it's not specified?

Reply:
That was my plan, even though I see that we don't do it for any other configuration parameter.
My rationale was that: 1) most users should not need it and setting the top-level `.snatFullyRandomPorts` parameter should be enough, 2) it will avoid confusion / issues if users search the parameter by name in the manifest and up setting `.egress.snatFullyRandomPorts` when they meant to set `.snatFullyRandomPorts`. What do you think?

Reply:
I feel it would become a bit contradictory: if we assume two options are useful and the feature requester needs to set both, they will only find one option in the published yaml: https://github.com/antrea-io/antrea/blob/ce833392aa4bc77464969345b7e3ef85da63728d/build/yamls/antrea.yml. Only helm users would know there are two options.

Reply:
I pushed a commit to stop hiding the field when it is nil / empty